review-notifications/src/background.js

```diff
@@ -90,6 +94,7 @@ function addPRToList(prList, newPR) {
     title: newPR.title,
     updated: newPR.updated,
     commentsData: newPR.commentsData,
+    requested_reviewers: newPR.requested_reviewers,
```

Comment:
Unless you have a good reason for this name stick to one convention. `requestedReviewers` instead of `requested_reviewers`